lib/framework/widget/camera_manager.dart

```diff
@@ -56,24 +22,37 @@ class CameraManager {
         ),
       );
     }
+    // if camera action is not null 
     else {
-      print('Check Else Condition');
+    // camera instance is created for set properties to setter
+    // if any other camera instance is created already automatically remove beacuse camera instance is dispose
+    // when screen is close
+     
       CameraScreen camera = CameraScreen();
       cameraAction.options!['mode'] == null
```

Comment:
This is wrong as you expect the user to enters the correct mode (they can enter mode=whatever and you will pick up the invalid value). Better to use an enum, something liek this:
if (options['mode'] == 'photo') 
  mode = CameraMode.PHOTO;
else if (options['model] == 'video')
   mode = CameraMode.VIDEO  
else
   mode = CameraMode.BOTH.
   
This way you always get a valid value, and default to BOTH (that's the spec) if they enter incorrect value